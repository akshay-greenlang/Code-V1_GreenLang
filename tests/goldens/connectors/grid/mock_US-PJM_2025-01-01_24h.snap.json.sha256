0003abcec832442d37074d427a526e21951dca2a864c1e2140f3b9f1523fc00f