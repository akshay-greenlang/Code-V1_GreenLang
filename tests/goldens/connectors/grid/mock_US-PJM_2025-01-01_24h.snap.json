{"connector_id":"grid/intensity/mock","connector_version":"0.1.0","payload":{"metadata":{"algorithm":"seasonal_sinusoid","connector":"grid/intensity/mock","data_points":24,"region_offset":175,"seed":"dbfa3697ef265636fd6b4870801c6996ff2949dfe1da41c2f4d2de8308fa9baf","version":"0.1.0"},"region":"US-PJM","resolution":"hour","series":[{"quality":"simulated","ts":"2025-01-01T00:00:00Z","unit":"gCO2/kWh","value":"475"},{"quality":"simulated","ts":"2025-01-01T01:00:00Z","unit":"gCO2/kWh","value":"526.8"},{"quality":"simulated","ts":"2025-01-01T02:00:00Z","unit":"gCO2/kWh","value":"575"},{"quality":"simulated","ts":"2025-01-01T03:00:00Z","unit":"gCO2/kWh","value":"616.4"},{"quality":"simulated","ts":"2025-01-01T04:00:00Z","unit":"gCO2/kWh","value":"648.2"},{"quality":"simulated","ts":"2025-01-01T05:00:00Z","unit":"gCO2/kWh","value":"668.2"},{"quality":"simulated","ts":"2025-01-01T06:00:00Z","unit":"gCO2/kWh","value":"675"},{"quality":"simulated","ts":"2025-01-01T07:00:00Z","unit":"gCO2/kWh","value":"668.2"},{"quality":"simulated","ts":"2025-01-01T08:00:00Z","unit":"gCO2/kWh","value":"648.2"},{"quality":"simulated","ts":"2025-01-01T09:00:00Z","unit":"gCO2/kWh","value":"616.4"},{"quality":"simulated","ts":"2025-01-01T10:00:00Z","unit":"gCO2/kWh","value":"575"},{"quality":"simulated","ts":"2025-01-01T11:00:00Z","unit":"gCO2/kWh","value":"526.8"},{"quality":"simulated","ts":"2025-01-01T12:00:00Z","unit":"gCO2/kWh","value":"475"},{"quality":"simulated","ts":"2025-01-01T13:00:00Z","unit":"gCO2/kWh","value":"423.2"},{"quality":"simulated","ts":"2025-01-01T14:00:00Z","unit":"gCO2/kWh","value":"375"},{"quality":"simulated","ts":"2025-01-01T15:00:00Z","unit":"gCO2/kWh","value":"333.6"},{"quality":"simulated","ts":"2025-01-01T16:00:00Z","unit":"gCO2/kWh","value":"301.8"},{"quality":"simulated","ts":"2025-01-01T17:00:00Z","unit":"gCO2/kWh","value":"281.8"},{"quality":"simulated","ts":"2025-01-01T18:00:00Z","unit":"gCO2/kWh","value":"275"},{"quality":"simulated","ts":"2025-01-01T19:00:00Z","unit":"gCO2/kWh","value":"281.8"},{"quality":"simulated","ts":"2025-01-01T20:00:00Z","unit":"gCO2/kWh","value":"301.8"},{"quality":"simulated","ts":"2025-01-01T21:00:00Z","unit":"gCO2/kWh","value":"333.6"},{"quality":"simulated","ts":"2025-01-01T22:00:00Z","unit":"gCO2/kWh","value":"375"},{"quality":"simulated","ts":"2025-01-01T23:00:00Z","unit":"gCO2/kWh","value":"423.2"}],"unit":"gCO2/kWh"},"provenance":{"connector_id":"grid/intensity/mock","connector_version":"0.1.0","metadata":{"data_points":24,"end":"2025-01-02T00:00:00+00:00","region":"US-PJM","resolution":"hour","start":"2025-01-01T00:00:00+00:00"},"mode":"record","query_hash":"ce52e240d3a3f64fb98972d5f5faafb7537ab96e1c79ed610a81bfc386a36f75","schema_hash":"2cf663a8a03d50acb0c1df2fba40940ffb4f63f0d1882a6386a70320e2bc06f6","seed":"dbfa3697ef265636fd6b4870801c6996ff2949dfe1da41c2f4d2de8308fa9baf","snapshot_id":"79502220ff3f2a3f8710b4c5c6c3afed438a34324e6afa31ea81daa37ab46cc5","timestamp":"2025-10-09T05:13:40.763671Z"},"schema":{"payload_model":"GridIntensityPayload","provenance_model":"ConnectorProvenance"}}